docker-compose.yaml

```diff
@@ -4,7 +4,7 @@ services:
     links:
       - localstack:localstack
   localstack:
-    image: localstack/localstack
+    image: localstack/localstack:2.3.2
```

Comment:
Weird one this one, I wouldn't push this up. I think this requires a spike, so the pipeline seems to pass with `localstack/localstack` - wouldn't want to risk shaking the pipeline before we can roll to pre-prod. Definitely tech debt again, I know it makes zero sense - me and Joe were baffled! Need to look at this after pen.